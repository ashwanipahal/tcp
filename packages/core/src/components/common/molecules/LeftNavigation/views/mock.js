const subnavigationMock = [
  {
    subSections: null,
    leafLink: {
      url: '/helpCenter',
      id: 'help-center-home',
      text: 'Help Center Home',
      displayName: 'Help Center Home',
      component: 'help-center-home',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: '/account/place-rewards',
      text: 'My Place Rewards',
      id: 'place-rewards',
      displayName: 'My Place Rewards',
      component: 'place-rewards',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: '/account/favorites',
      text: 'My Place Rewards Program',
      id: 'my-place-rewards-program',
      displayName: 'My Place Rewards Program',
      component: 'my-place-rewards-program',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: '/account/extra-points',
      text: 'Earn Extra Points',
      id: 'extra-points',
      displayName: 'Earn Extra Points',
      component: 'extra-points',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: '/account/favorites',
      text: 'My Place Rewards Credit Card',
      id: 'my-place-rewards-credit-card',
      displayName: 'My Place Rewards Credit Card',
      title: 'my-place-rewards-credit-card',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Coupons and Promotions',
      id: 'coupons-and-promotions',
      displayName: 'Coupons and Promotions',
      component: 'coupons-and-promotions',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Help With My Order',
      id: 'help-with-my-order',
      displayName: 'Help With My Order',
      component: 'help-with-my-order',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Online Ordering',
      id: 'online-ordering',
      displayName: 'Online Ordering',
      component: 'online-ordering',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Policies',
      id: 'policies',
      displayName: 'Policies',
      component: 'policies',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Terms and Conditions',
      id: 'terms-and-conditions',
      displayName: 'Terms and Conditions',
      component: 'terms-and-conditions',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Mobile Terms and Conditions',
      id: 'mobile-terms-and-conditions',
      displayName: 'Mobile Terms and Conditions',
      component: 'mobile-terms-and-conditions',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Shipping to the U.S.',
      id: 'shipping-to-the-U.S.',
      displayName: 'Shipping to the U.S.',
      component: 'shipping-to-the-U.S.',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Tools',
      id: 'tools',
      displayName: 'Tools',
      component: 'tools',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
  {
    subSections: null,
    leafLink: {
      url: null,
      text: 'Contact Us',
      id: 'contact-us',
      displayName: 'Contact Us',
      component: 'contact-us',
      target: '',
      external: 0,
      __typename: 'Link',
    },
    __typename: 'HelpCenterNavigationContent',
  },
];

export default subnavigationMock;
