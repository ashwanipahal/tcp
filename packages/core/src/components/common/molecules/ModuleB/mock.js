/* eslint-disable sonarjs/no-duplicate-string */

export default {
  moduleB: {
    contentId: 'f1733fc9-6db0-4042-9844-99980420359f',
    name: 'moduleB',
    type: 'module',
    composites: {
      largeCompImage: [
        {
          headerText: [
            {
              textItems: [
                {
                  text: 'NEW WASHES',
                  style: 'style1',
                },
                {
                  text: 'MORE FITS',
                  style: 'style1',
                },
                {
                  text: 'SLIM, PLUS & HUSKY',
                  style: 'style2',
                },
                {
                  text: 'SIZES UPTO 18!',
                  style: 'style1',
                },
              ],
              link: {
                url: '/tees',
                text: '',
                title: '',
                target: '',
                external: 0,
                action: '',
              },
              icon: {
                icon: '',
                placement: '',
              },
            },
          ],
          promoBanner: [
            {
              link: {
                url: '/banner/url',
                text: '',
                title: '',
                target: '',
                external: 0,
                action: '',
              },
              textItems: [
                {
                  text: 'ALL BASIC DENIM',
                  style: 'overlay_banner_header',
                },
                {
                  text: '$ 7 80 &UP',
                  style: 'overlay_promo_banner_style',
                },
              ],
            },
          ],
          linkedImage: [
            {
              image: {
                url:
                  'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
                alt: 'Graphic Tees',
                title: 'Graphic Tees',
                crop_d: 'c_crop,g_face:center,q_auto:best,w_962',
                crop_t: 'c_crop,g_face:center,q_auto:best,w_962',
                crop_m: 'c_crop,g_face:center,q_auto:best,w_962',
              },
              link: {
                url: '/node/product/<uuid>',
                text: 'Graphic Tees',
                title: 'Graphic Tees',
                target: '',
                external: 0,
                action: '',
              },
            },
          ],
        },
      ],
      ctaItems: [
        {
          button: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            text: 'Girl',
            title: 'Girl',
            target: '',
            external: 0,
            action: '',
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            alt: 'Graphic Tees',
            title: 'Graphic Tees',
            crop_d: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_t: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_m: 'c_crop,g_face:center,q_auto:best,w_962',
          },
        },
        {
          button: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            text: 'Toddler Girl',
            title: 'Toddler Girl',
            target: '',
            external: 0,
            action: '',
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            alt: 'Graphic Tees',
            title: 'Graphic Tees',
            crop_d: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_t: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_m: 'c_crop,g_face:center,q_auto:best,w_962',
          },
        },
        {
          button: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            text: 'Boy',
            title: 'Boy',
            target: '',
            external: 0,
            action: '',
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            alt: 'Graphic Tees',
            title: 'Graphic Tees',
            crop_d: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_t: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_m: 'c_crop,g_face:center,q_auto:best,w_962',
          },
        },
        {
          button: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            text: 'Toddler Boy',
            title: 'Toddler Boy',
            target: '',
            external: 0,
            action: '',
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            alt: 'Graphic Tees',
            title: 'Graphic Tees',
            crop_d: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_t: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_m: 'c_crop,g_face:center,q_auto:best,w_962',
          },
        },
        {
          button: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            text: 'Baby',
            title: 'Baby',
            target: '',
            external: 0,
            action: '',
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/q_auto:best/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            alt: 'Graphic Tees',
            title: 'Graphic Tees',
            crop_d: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_t: 'c_crop,g_face:center,q_auto:best,w_962',
            crop_m: 'c_crop,g_face:center,q_auto:best,w_962',
          },
        },
      ],
    },
    submodules: {},
    set: [
      {
        key: 'moduleWidth',
        val: 'half',
      },
      {
        key: 'ctaType',
        val: 'divImageCTACarousel',
      },
      {
        key: 'bannerPostition',
        val: 'overlay',
      },
    ],
  },
};
