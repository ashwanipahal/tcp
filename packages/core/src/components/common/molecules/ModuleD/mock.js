const plpUrl =
  'https://www.childrensplace.com/us/c/girls-kids-shorts?icid=hp_s3a_imagenbutton_g_050519_shorts';
const fmt = 'Family Tees';
const mmt = 'Mom & Me Tees';
export default {
  moduleD: {
    contentId: '73f6a699-79a4-4874-994f-ab306dd66dca',
    name: 'moduleD',
    type: 'module',
    composites: {
      headerText: {
        textLines: {
          text: 'Mini me shop',
          color: 'color2',
          style: 'style1',
        },
        link: {
          url: plpUrl,
          title: '',
          target: '',
        },
      },
      promoBanner: {
        items: [
          {
            link: {
              url: plpUrl,
              title: '',
              target: '',
              external: 0,
            },
            image: {
              url: '',
              alt: 'alt 1',
              title: 'Image link text',
            },
          },
        ],
      },
      smallCompImage: {
        items: [
          {
            link: {
              url: plpUrl,
              title: fmt,
              target: '',
              external: 0,
            },
            image: {
              url:
                'https://res.cloudinary.com/tcp-dam-test/image/upload/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
              title: fmt,
              alt: fmt,
            },
          },
          {
            link: {
              url: plpUrl,
              title: mmt,
              target: '',
              external: 0,
            },
            image: {
              url:
                'https://res.cloudinary.com/tcp-dam-test/image/upload/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME3_vmfhnu.jpg',
              title: mmt,
              alt: mmt,
            },
          },
          {
            link: {
              url: plpUrl,
              title: fmt,
              target: '',
              external: 0,
            },
            image: {
              url:
                'https://res.cloudinary.com/tcp-dam-test/image/upload/v1558543114/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME4_iuzwmp.jpg',
              title: fmt,
              alt: fmt,
            },
          },
          {
            link: {
              url: plpUrl,
              title: mmt,
              target: '',
              external: 0,
            },
            image: {
              url:
                'https://res.cloudinary.com/tcp-dam-test/image/upload/v1558543114/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME2_uwtbdd.jpg',
              title: mmt,
              alt: mmt,
            },
          },
        ],
      },
      singleCTAButton: {
        url: plpUrl,
        target: '',
        title: 'SHOP ALL',
        external: 0,
      },
    },
  },
};
