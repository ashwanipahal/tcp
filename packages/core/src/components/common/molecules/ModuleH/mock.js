export default {
  moduleH: {
    contentId: 'c755fe7e-5811-4039-ba96-7efc0a2d2d64',
    name: 'moduleH',
    type: 'module',
    composites: {
      headerText: {
        textLines: [
          {
            text: 'New arrivals',
            color: 'color1',
            style: 'style1',
          },
          {
            text: 'New adventures',
            color: 'color1',
            style: 'style1',
          },
        ],
      },
      divCTALinks: [
        {
          link: {
            url: '/toddler',
            title: '',
            target: '',
            external: 0,
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/c_crop,g_face:center,h_541,w_1440/v1555526246/samples/landscapes/nature-mountains.jpg',
            title: 'Toddler girl',
            alt: 'Toddler girl',
          },
          styled: {
            text: 'Toddler Girl',
            color: 'color1',
            style: 'style1',
          },
        },
        {
          link: {
            url: '/girl',
            title: '',
            target: '',
            external: 0,
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/c_fill,g_face,h_541,w_1440/v1558543115/ecom/assets/content/tcp/us/home/moduled/US-HP-050519-MINIME1_h9cwcd.jpg',
            title: 'Girl',
            alt: 'Girl',
          },
          styled: {
            text: 'Girl',
            color: 'color2',
            style: 'style2',
          },
        },
        {
          link: {
            url: '/toddler-boy',
            title: '',
            target: '',
            external: 0,
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/c_fill,g_west,h_541,w_1440/v1555526244/samples/ecommerce/leather-bag-gray.jpg',
            title: 'Toddler boy',
            alt: 'Toddler boy',
          },
          styled: {
            text: 'Toddler boy style',
            color: 'color3',
            style: 'style3',
          },
        },
        {
          link: {
            url: '/boy',
            title: '',
            target: '',
            external: 0,
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/c_crop,g_north_west,h_541,w_1440/v1555526245/samples/food/spices.jpg',
            title: 'Boy',
            alt: 'Boy',
          },
          styled: {
            text: 'Boy',
            color: 'color4',
            style: 'style4',
          },
        },
        {
          link: {
            url: '/baby',
            title: '',
            target: '',
            external: 0,
          },
          image: {
            url:
              'https://res.cloudinary.com/tcp-dam-test/image/upload/c_scale,h_541,w_1440/v1555526249/samples/landscapes/landscape-panorama.jpg',
            title: 'Baby',
            alt: 'Baby',
          },
          styled: {
            text: 'Baby',
            color: 'color4',
            style: 'style5',
          },
        },
      ],
    },
  },
};
