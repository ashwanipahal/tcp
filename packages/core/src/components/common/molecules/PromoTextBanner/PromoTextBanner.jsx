// @flow
import React from 'react';
import { BodyCopy } from '../../atoms';
import withStyles from '../../hoc/withStyles';
import PromoBannerStyle from './PromoTextBanner.style';

type Props = {
  textItems: Object,
  className: String,
};

/**
 * This component produces a Promo Text banner
 * Expects textItems array consisting of objects in below format
 * {
 *    style: "",
 *    text: ""
 * }
 * This component uses BodyCopy atom and accepts all properties of BodyCopy
 * @param {*} props
 */
const PromoBanner = (props: Props) => {
  const { textItems, className, ...otherProps } = props;

  return (
    <BodyCopy component="div" className={className} {...otherProps}>
      <React.Fragment>
        {textItems.map(({ text, style }, index) => (
          <span className={`promo-text ${style}`}>{index ? ` ${text}` : text}</span>
        ))}
      </React.Fragment>
    </BodyCopy>
  );
};

export default withStyles(PromoBanner, PromoBannerStyle);
