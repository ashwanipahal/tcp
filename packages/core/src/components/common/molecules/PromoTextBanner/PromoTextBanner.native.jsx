// @flow
import React from 'react';

type Props = {
  promoTextBanner: Array<Object>,
  bodyCopyStyles: Array<Object>,
};

/**
 * This component produces a Promo Text banner
 * Expects textItems array consisting of objects in below format
 * {
 *    style: "",
 *    text: ""
 * }
 * This component accepts BodyCopy styles in array return matching BodyCopy style with
 * the key provided by CMS
 * @param {*} props
 */
const PromoTextBanner = (props: Props) => {
  const {
    promoTextBanner: [{ textItems }],
    /* bodyCopyStyles is a array of BodyCopy component with key of style1,style2,style3 etc.
    The keys are coming from CMS */
    bodyCopyStyles,
    ...otherProps
  } = props;
  return [
    textItems.map(({ text, style }, index) => {
      const StyleBodyCopy = bodyCopyStyles[style];
      return <StyleBodyCopy text={index ? ` ${text}` : text} {...otherProps} />;
    }),
  ];
};

export default PromoTextBanner;
