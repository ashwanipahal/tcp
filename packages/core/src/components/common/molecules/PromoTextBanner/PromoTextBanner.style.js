import { css } from 'styled-components';

export default css`
  text-align: center;
  .promo-text {
    display: block;
    @media ${props => props.theme.mediaQuery.medium} {
      display: inline;
    }
  }
  .moduleN-promoBanner-text1 {
    display: inline-block;
    font-size: 68px;
    font-weight: ${props => props.theme.typography.fontWeights.black};
    padding: 0 1% 0 11%;

    @media ${props => props.theme.mediaQuery.medium} {
      font-size: 74px;
      padding-left: 5%;
    }

    @media ${props => props.theme.mediaQuery.large} {
      font-size: 74px;
      padding: inherit;
    }
  }
  .moduleN-promoBanner-text2 {
    display: inline-block;
    font-size: 46px;
    vertical-align: top;
    font-weight: ${props => props.theme.typography.fontWeights.black};

    @media ${props => props.theme.mediaQuery.large} {
      font-size: 74px;
      margin-right: 5px;
    }
  }
  .moduleN-promoBanner-text3 {
    display: inline-block;
    font-size: 20px;
    font-weight: ${props => props.theme.typography.fontWeights.black};
    position: relative;
    right: 11%;

    @media ${props => props.theme.mediaQuery.medium} {
      right: 6%;
    }

    @media ${props => props.theme.mediaQuery.large} {
      font-size: 74px;
      margin-right: 9px;
      position: inherit;
    }
  }
  .moduleN-promoBanner-text4 {
    display: block;
    font-size: 16px;
    margin-bottom: 18px;

    @media ${props => props.theme.mediaQuery.medium} {
      margin-bottom: 18px;
    }

    @media ${props => props.theme.mediaQuery.large} {
      display: inline-block;
      margin-bottom: 35px;
      font-size: 26px;
      width: 0;
      position: inherit;
    }
  }
`;
