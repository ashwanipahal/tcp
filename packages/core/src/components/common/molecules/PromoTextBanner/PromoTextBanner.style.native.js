import styled from 'styled-components';

import BodyCopy from '../../atoms/BodyCopy';

const StyledBodyCopy = styled(BodyCopy)`
  ${({ lineHeight }) => (lineHeight ? `line-height: ${lineHeight}` : '')}
`;

export { StyledBodyCopy as BodyCopy };

export default {
  StyledBodyCopy: BodyCopy,
};
