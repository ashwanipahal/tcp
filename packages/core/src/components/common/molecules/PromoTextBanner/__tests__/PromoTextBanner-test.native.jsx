import React from 'react';
import { shallow } from 'enzyme';
import mock from '../../../../../services/abstractors/common/moduleK/mock';
import PromoTextBanner from '../PromoTextBanner.native';

const bodyCopyStyles = {
  style1: () => <div>Style1</div>,
  style2: () => <div>Style2</div>,
  style3: () => <div>Style3</div>,
};

describe('PromoTextBanner native component', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <PromoTextBanner
        promoTextBanner={mock.moduleK.composites.masonryGrid[0].promoTextBanner}
        bodyCopyStyles={bodyCopyStyles}
        dataLocator="moduleK_text_"
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
