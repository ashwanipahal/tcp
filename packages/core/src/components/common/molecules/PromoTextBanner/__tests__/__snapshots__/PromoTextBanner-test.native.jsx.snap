// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PromoTextBanner native component renders correctly 1`] = `
Array [
  <style2
    dataLocator="moduleK_text_"
    text="ALL DINOS"
  />,
  <style3
    dataLocator="moduleK_text_"
    text=" 50% OFF"
  />,
]
`;
